src/ray/gcs/tables.cc

```diff
@@ -38,6 +40,8 @@ namespace gcs {
 template <typename ID, typename Data>
 Status Log<ID, Data>::Append(const JobID &job_id, const ID &id,
                              std::shared_ptr<DataT> &dataT, const WriteCallback &done) {
+  RAY_LOG(DEBUG) << client_->client_table().GetLocalClientId() << " Append to table "
+                 << EnumNameTablePrefix(prefix_) << " key " << id;
```

Comment:
@stephanie-wang do we still want to keep these logs? I'm not sure if they will be too verbose.

Reply:
Thanks, yeah, I'll delete these!